var/spack/repos/builtin/packages/intel-oneapi-compilers-classic/package.py

```diff
@@ -42,9 +46,22 @@ def setup_run_environment(self, env):
 
         and from setting CC/CXX/F77/FC
         """
-        bin = join_path(self.spec['intel-oneapi-compilers'].prefix,
-                        'compile', 'linux', 'bin', 'intel64')
-        env.set('CC', join_path(bin, 'icc'))
-        env.set('CXX', join_path(bin, 'icpc'))
-        env.set('F77', join_path(bin, 'ifort'))
-        env.set('FC', join_path(bin, 'ifort'))
+        env.set('CC', join_path(self.oneapi_compiler_prefix.bin, 'icc'))
```

Comment:
```suggestion
        env.set('CC', join_path(self.oneapi_compiler_prefix.bin.intel64, 'icc'))
```

Don't you need the intel64?
```
rscohn1@rscohn1-mobl1:~$ ls /opt/intel/oneapi/compiler/latest/linux/bin/intel64
codecov  icc.cfg  icpc.cfg         map_opts  profdcg    profmergesampling  tselect  xiar.cfg  xild.cfg
icc      icpc     libcilkrts.so.5  mcpcom    profmerge  proforder          xiar     xild
rscohn1@rscohn1-mobl1:~$
```

Reply:
Yes, thank you